.template/variants/web/package.json.rb

```diff
@@ -21,13 +23,15 @@
 run 'yarn add --dev postcss@8.4.5'
 
 # Setup scripts
-source_stylesheet = "./app/assets/stylesheets/application.scss"
-bundled_stylesheet = "./app/assets/builds/application.css"
-
 run 'npm set-script eslint "eslint . --color"'
 run 'npm set-script eslint:fix "eslint . --color --fix"'
 run 'npm set-script stylelint "stylelint **/*.scss --color"'
 run 'npm set-script stylelint:fix "stylelint **/*.scss --color --fix"'
 run 'npm set-script lint "yarn eslint && yarn stylelint"'
 run 'npm set-script lint:fix "yarn eslint:fix && yarn stylelint:fix"'
```

Comment:
Update this to `codebase` and `codebase.fix` to be consistent to other templates

Reply:
Updated in 61f5cfa